rules/no-unknown-modifiers.js

```diff
@@ -0,0 +1,46 @@
+'use strict';
+var createAvaRule = require('../create-ava-rule');
+
+var modifiers = {
+	after: true,
+	afterEach: true,
+	before: true,
+	beforeEach: true,
+	cb: true,
+	only: true,
+	serial: true,
+	skip: true,
+	todo: true
+};
```

Comment:
Why not just an array? You won't gain any noticeable speed improvement for such a small list.